core/src/main/java/hudson/cli/handlers/JobOptionHandler.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2004-2009, Sun Microsystems, Inc.
```

Comment:
"The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software."

Since this file is essentially a rename I thought it was appropriate to leave it intact (the dates were never changed in other revisions).

If I had written it from scratch I never would have put a copyright/license on it at all (as in TopLevelItemOptionHandler.java).  I can take it off completely if that's better.